dwave_networkx/algorithms/partitioning.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2021 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+from dwave_networkx import chimera_graph
+from dimod import DiscreteQuadraticModel
+from dwave_system import LeapHybridDQMSampler
```

Comment:
Should be `dwave.system`, but looks like you can just remove this import, because `LeapHybridDQMSampler` is not being used, right?